tests/nextflow-tests/run_tests_simple_archive.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Might as well [run this in actions](https://github.com/EBIvariation/eva-submission/blob/master/.github/workflows/python-package.yml#L67) too